### PR TITLE
bpo-1635741: Port _bisect module to multi-phase init

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue1635741

### CODE REVIEW

**Comment by corona10** [comment]
@shihai1991 Can you please take a look?

File: Misc/NEWS.d/next/Core and Builtins/2020-09-26-14-43-30.bpo-1635741.aJS9B3.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Port :mod:`_bisect` to the multi-phase initialization (:pep:`489`).
```
Comment (vstinner) [inline Misc/NEWS.d/next/Core and Builtins/2020-09-26-14-43-30.bpo-1635741.aJS9B3.rst]:
```suggestion
Port the :mod:`_bisect` module to the multi-phase initialization API (:pep:`489`).
```

File: Modules/_bisectmodule.c [hunk 2]
```diff
@@ -234,21 +234,21 @@ having to sort the list after each insertion. For long lists of items with\n\
...
-    -1,
-    bisect_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
+    .m_name = "_bisect",
+    .m_doc = module_doc,
+    .m_methods = bisect_methods,
+    .m_slots = _bisectmodule_slots,
```
Comment (vstinner) [inline Modules/_bisectmodule.c]: If there is no slot, you can leave m_slots as NULL.

**Review by shihai1991 — APPROVED** [review]
LGTM :)

File: Modules/_bisectmodule.c (hunk 2 above)
Comment (corona10) [reply Modules/_bisectmodule.c]: Thanks!

**Review by vstinner — APPROVED** [review]
LGTM. This conversion was easy :-D

I checked manually: the module and the module functions are stateless, there is no other change needed.

**Pull request merged**